members/CAL000397.yaml

```diff
@@ -39,6 +39,11 @@ contact_form:
           options:
             max_length: 2000
           required: true
+    - javascript:
+        - name: number-replace
+          values: ["$NAME_LAST"]
+          selectors: ["#txtLastName"]
+          commands: ["elements[0].value = values[0].replace(/\d/g,'')"]
```

Comment:
this replaces numbers?

Reply:
Si. Looks for any number in the last name input and replaces it with a blank space

Reply:
"Got a long list of numbers, who'll fail to validate,

Got a blank space baby, I'll post your name"